lib/std/os/posix.zig

```diff
@@ -0,0 +1,829 @@
+//! This file contains posix abstractions and wrappers around
+//! OS-specific APIs for conforming platforms, whether libc is or is not linked.
+//! Purpose is to have error handling for APIs, which can not have comparable
+//! semantics on non-posix systems like Windows.
+//! Reasons are besides design failures (signaling, process management) more
```

Comment:
"design failures" is very strong phrasing. Not sure, how I should phrase it better.

Reply:
fixed.